README.md

```diff
@@ -1,38 +1,45 @@
+[![Conda Forge](https://img.shields.io/conda/vn/conda-forge/autogluon.svg)](https://anaconda.org/conda-forge/autogluon)
 [![Python Versions](https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C%203.10%20%7C%203.11-blue)](https://pypi.org/project/autogluon/)
-[![GitHub license](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](./LICENSE)
 [![Downloads](https://pepy.tech/badge/autogluon/month)](https://pepy.tech/project/autogluon)
-[![](https://img.shields.io/discord/1043248669505368144?logo=discord&style=flat)](https://discord.gg/wjUmjqAc2N)
+[![GitHub license](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](./LICENSE)
+[![Discord](https://img.shields.io/discord/1043248669505368144?logo=discord&style=flat)](https://discord.gg/wjUmjqAc2N)
 [![Twitter](https://img.shields.io/twitter/follow/autogluon?style=social)](https://twitter.com/autogluon)
+[![Continuous Integration](https://github.com/autogluon/autogluon/actions/workflows/continuous_integration.yml/badge.svg)](https://github.com/autogluon/autogluon/actions/workflows/continuous_integration.yml)
+[![Platform Tests](https://github.com/autogluon/autogluon/actions/workflows/platform_tests-command.yml/badge.svg?event=schedule)](https://github.com/autogluon/autogluon/actions/workflows/platform_tests-command.yml)
 
-[Install Instructions](https://auto.gluon.ai/stable/install.html) | Documentation ([Stable](https://auto.gluon.ai/stable/index.html) | [Latest](https://auto.gluon.ai/dev/index.html))
+[Install Instructions](https://auto.gluon.ai/stable/install.html) | [Documentation](https://auto.gluon.ai/stable/index.html) | [Release Notes](https://auto.gluon.ai/stable/whats_new/index.html)
 
 AutoGluon automates machine learning tasks enabling you to easily achieve strong predictive performance in your applications.  With just a few lines of code, you can train and deploy high-accuracy machine learning and deep learning models on image, text, time series, and tabular data.
+</div>
+
+## 💾 Installation
+
+AutoGluon is supported on Python 3.8 - 3.11 and is available on Linux, MacOS, and Windows OS.
 
-## Example
+You can install AutoGluon with:
 
 ```python
-# First install package from terminal:
-# pip install -U pip
-# pip install -U setuptools wheel
-# pip install autogluon  # autogluon==1.0.0
-
-from autogluon.tabular import TabularDataset, TabularPredictor
-train_data = TabularDataset('https://autogluon.s3.amazonaws.com/datasets/Inc/train.csv')
-test_data = TabularDataset('https://autogluon.s3.amazonaws.com/datasets/Inc/test.csv')
-predictor = TabularPredictor(label='class').fit(train_data, time_limit=120)  # Fit models for 120s
-leaderboard = predictor.leaderboard(test_data)
+pip install autogluon
+```
+
+Visit our [Installation Guide](https://auto.gluon.ai/stable/install.html) for detailed instructions.
+
+## :zap: Quickstart
+
+Solve ML tasks end-to-end in just 3 lines of code!
```

Comment:
What about "Build accurate end-to-end ML models in just 3 lines of code!"? I'm not sure what exactly solving ML tasks end-to-end means here.

Reply:
Nice idea! Updated to this wording